modules/configuring-the-lora-training-job.adoc

```diff
@@ -0,0 +1,249 @@
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#creating-a-data-science-project_projects[Creating a data science project].
+endif::[]
+
+* You have Admin access for the data science project.
+** If you created the project, you automatically have Admin access. 
+** If you did not create the project, your cluster administrator must give you Admin access.
+
+ifndef::upstream[]
+* If Kueue is not installed in your environment, you have configured the Training Operator permissions as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/tuning-a-model-by-using-the-training-operator_distributed-workloads/#configuring-the-training-operator-permissions-when-not-using-kueue_distributed-workloads[Configuring the Training Operator permissions when not using Kueue].
+endif::[]
+ifdef::upstream[]
+* If Kueue is not installed in your environment, you have configured the Training Operator permissions as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-the-training-operator-permissions-when-not-using-kueue_distributed-workloads[Configuring the Training Operator permissions when not using Kueue].
+endif::[]
+
+* You have access to a model.
+* You have access to data that you can use to train the model.
+
+.Procedure
+. In a terminal window, if you are not already logged in to your OpenShift cluster, log in to the OpenShift CLI as shown in the following example:
++
+[source,subs="+quotes"]
+----
+$ oc login __<openshift_cluster_url>__ -u __<username>__ -p __<password>__
+----
+
+. Configure a LoRA training job, as follows:
+.. Create a YAML file named `lora-config.yaml`.
+.. Add the following `ConfigMap` object definition:
++
+[source]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: lora-config
+  namespace: kfto
+data:
+  lora_config.json: |
+    {
+      "model_name_or_path": "bigscience/bloom-560m",
+      "training_data_path": "/data/input/twitter_complaints.json",
+      "output_dir": "/data/output/tuning/bloom-twitter",
+      "num_train_epochs": 10.0,
+      "per_device_train_batch_size": 4,
+      "gradient_accumulation_steps": 4,
+      "learning_rate": 1e-05,
+      "response_template": "\n### Label:",
+      "dataset_text_field": "output",
+      "use_flash_attn": false,
+      "lora_r": 8,
+      "lora_alpha": 8,
+      "lora_dropout": 0.1,
+      "bias": "none",
+      "target_modules": ["q_proj", "v_proj"]
+    }
+
+----
+.. Edit the training job metadata as shown in the following table.
++
+.Training job metadata
+[cols="16,84"]
+|===
+|Parameter | Value
+
+|`name`
+|Name of the training job
+
+|`namespace`
+|Name of your project
+|===
+
+.. Edit the parameters of the training job as shown in the following table.
++
+.Training job parameters
+[cols="30,70"]
+|===
+|Parameter | Value
+
+|`model_name_or_path`
+|Name of the pre-trained model
```

Comment:
Name of the model taken from HuggingFace webpage or path to the model in training job container.